packages/devextreme/js/localization/messages/fi.json

```diff
@@ -542,6 +542,8 @@
         "dxHtmlEditor-aiTryAgain": "Try again",
         "dxHtmlEditor-aiGenerate": "Generate",
         "dxHtmlEditor-aiStop": "Stop",
+        "dxHtmlEditor-aiToolbarItem": "AI Assistant toolbar item",
+        "dxHtmlEditor-aiResultTextArea": "AI Assistant result",
```

Comment:
afair we usually explicitly specify `label` in the variable name. `aiResultTextArea` looks strange for me. Please discuss with TW

Reply:
Okay let's use the label postfix.

Reply:
[7b02fd072bb8f2ac3edcddf1512018fba3ec867c]